Source/plugins/IPlugin.h

```diff
@@ -223,6 +223,14 @@ namespace PluginHost {
         virtual ISecurity* Officer(const string& token) = 0;
     };
 
+    struct EXTERNAL IConfigure : virtual public Core::IUnknown {
```

Comment:
There is already an IConfigure you can use for the plugins, no need to define a new one that is part of the core. See interfaces/IConfigure..

Reply:
Sure will use that from interfaces

Reply:
done